ceph-salt-formula/salt/ceph-salt/apply/cephconfigure.sls

```diff
@@ -2,14 +2,10 @@
 
 {% if 'cephadm' in grains['ceph-salt']['roles'] or 'admin' in grains['ceph-salt']['roles'] %}
 
-{{ macros.begin_stage('Find an admin host') }}
-
 find an admin host:
   ceph_orch.set_admin_host:
     - failhard: True
 
-{{ macros.end_stage('Find an admin host') }}
-
```

Comment:
I'm not sure I agree to drop the "Find an admin host" begin_stage and end_stage macro calls . . . . ceph-salt might spend a lot of time (up to 1800 seconds) in this loop and without the begin_stage message it will be less clear what the minions are doing.

Reply:
Explained in my previous comment.